Tests/FluentAssertions.Specs/Primitives/ObjectAssertionSpecs.cs

```diff
@@ -785,47 +808,57 @@ public void When_its_own_type_instance_and_asserting_not_assignable_it_should_fa
         {
             // Arrange
             var someObject = new DummyImplementingClass();
-            Action act = () => someObject.Should().NotBeAssignableTo(typeof(DummyImplementingClass), "because we want to test the failure {0}", "message");
+            Action act = () =>
+                someObject.Should().NotBeAssignableTo(typeof(DummyImplementingClass), "because we want to test the failure {0}",
+                    "message");
```

Comment:
Seems ReSharper's splitting algorithm has a [greedy](https://en.wikipedia.org/wiki/Greedy_algorithm) splitting approach? 

```cs
Action act = () => someObject.Should().NotBeAssignableTo(typeof(DummyImplementingClass), 
	"because we want to test the failure {0}", "message");
```

Reply:
No idea what you're trying to say with that 🤷‍♂️

Reply:
"Greedy" is a term in CS where an algorithm picks what immediately seems as the best choice, but in the larger picture turns out not to be optimal.

For this case I wanted to imply that splitting on `=>` and betwen `because` and `becauseArgs` is not optimal.
Following the the 130-rule the code, if we split after the type argument, we get two lines of instead of three lines ReSharper suggests.
Aesthetically I also prefer having `because` and `becauseArgs` closer.

Reply:
Me too, but that should be a case-by-case decision. The tool is not smart enough to think like us.

Reply:
For the aesthetics I agree it can't easily think like us, but if the objective is to do as few line splits to satisfy the 130-rule it's not impossible, it's just be more expensive to compute. (I'll stop the CS rambling now)

Reply:
After the merge, I'm going to make it a hard rule so that authors are forced to think about this.